libs/autofile/group.go

```diff
@@ -146,7 +146,7 @@ func (g *Group) OnStart() error {
 func (g *Group) OnStop() {
 	g.ticker.Stop()
 	if err := g.FlushAndSync(); err != nil {
-		g.Logger.Error("Error flushin to disk", "err", err)
+		g.Logger.Error("Error flushing to disk", "err", err)
```

Comment:
Nit: There is another "flushin" in line 163.
Also, might be worth upstreaming. The tendermint team moved the file to: https://github.com/tendermint/tendermint/blob/4334135651510b8586380cf86130143c2bbd5ec3/internal/libs/autofile/group.go#L149-L163